packages/nodes-base/nodes/Xero/InvoiceDescription.ts

```diff
@@ -115,7 +115,7 @@ export const invoiceFields = [
 		description: 'Contact ID',
 	},
 	{
-		description: 'Line Items Array (JSON)',
+		displayName: 'Line Items Array (JSON)',
```

Comment:
These are the types of fixes that should have been caught before committing it and deploying it. Even if we can't test it locally, you should have your editor set up to show syntax errors like this, and the build should succeed.